orcid-model/src/main/java/org/orcid/jaxb/model/record/summary_rc2/PeerReviewSummary.java

```diff
@@ -37,7 +37,7 @@
 import org.orcid.jaxb.model.record_rc2.ExternalIDs;
 
 @XmlAccessorType(XmlAccessType.FIELD)
-@XmlType(propOrder = { "createdDate", "lastModifiedDate", "source", "externalIdentifiers", "organization", "groupId", "completionDate" })
+@XmlType(propOrder = { "createdDate", "lastModifiedDate", "source", "externalIdentifiers", "completionDate", "groupId", "organization" })
```

Comment:
Hey @rcpeters, I dont know what you mean, display-index is an attribute(Already in the PeerReviewSummary object), and it don't affect the way we display the elements inside the peer review summary.